server/services/educationAndWorkPlanService.ts

```diff
@@ -92,6 +94,11 @@ export default class EducationAndWorkPlanService {
     return this.educationAndWorkPlanClient.unarchiveGoal(unarchiveGoalDto.prisonNumber, unarchiveGoalRequest, token)
   }
 
+  async completeGoal(completeGoalDto: CompleteGoalDto, token: string): Promise<unknown> {
```

Comment:
As above, we could do with a unit test for this; lets back fill it in another PR 👍